composer.json

```diff
@@ -54,7 +54,7 @@
         "drupal/admin_login_path": "^2.0",
         "drupal/admin_toolbar": "^3.0",
         "drupal/allowed_formats": "^2.0",
-        "drupal/anchor_link": "^2.4",
+        "drupal/anchor_link": "3.0.x-dev@dev",
```

Comment:
This is definitely not stable enough to rely on and i definitely don't like the idea of the composer merge. please remove all anchor link functionality until that module has a more stable release such as a alpha or beta release.